imagelab/DisplayImage.java

```diff
@@ -19,7 +19,6 @@
     private Image img;
     /** The display panel of this frame. */
     private DisPanel pane;
-
```

Comment:
What is the rationale for removing this whitespace?
Looking at this block of code, it seems like more (rather than less) whitespace might be useful.